File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/typescript/ADV-02_R.js
"use strict";
console.log("ADV-START");
const lines = require("fs").readFileSync(0, "utf8").split("\n");
const a = Number.parseInt(lines[0], 10);
const sum = a + 3;
console.log("OBS=V:" + sum);
console.log("ADV-END");

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/go/ADV-02_R.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	fmt.Println("ADV-START")
	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()
	p, _ := strconv.ParseInt(sc.Text(), 10, 64)
	a := int(p)
	v := a + 3
	fmt.Printf("OBS=V:%d\n", v)
	fmt.Println("ADV-END")
}
